package.json

```diff
@@ -1631,6 +1631,12 @@
             }
           }
         },
+        "cmake.additionalKits": {
+          "type": "array",
+          "default": [],
```

Comment:
It looks like my other comment about adding the `"items"` property here was resolved by accident.  We still need to provide stronger validation here.

Reply:
Oh right, I missed that.